src/app/core/entity/schema/entity-schema.service.ts

```diff
@@ -97,7 +97,7 @@ export class EntitySchemaService {
     for (const key of schema.keys()) {
       const schemaField: EntitySchemaField = schema.get(key);
 
-      if (data[key] === undefined) {
+      if (data[key] === undefined || data[key] === null) {
```

Comment:
Didn't we have discussions around this before, whether `null` being distinct from `undefined` might be a valid value in some special cases and therefore shouldn't be replaced by the default value?

Anyway ... let's do it like this then - and come back to it if it ever causes bugs